online_meetings/canvas_zoom_meetings.py

```diff
@@ -0,0 +1,138 @@
+# Script to get all sites where Zoom is visible and retrieve the meetings to generate a report
```

Comment:
If this makes sense to you, can this folder `online_meetings` have a python package called `zoom` and underneath that goes the `canvas_zoom_meetings.py` and `reports.py`. I see the `report.py` has zoom specific content so going in the package would make it organized

Reply:
I think the idea was we would also put BlueJeans stuff in here. I believe what we saw from the IQ folks suggested they were going to combine Zoom and BJ data into one table (similar properties to be tracked), but I could be mistaken. I think `online_meetings` makes sense.

Reply:
Yeah, I like the idea of having a single package for zoom and combining all of the zoom code. These are separate because Zhen and I wrote these and I didn't combine the code.